### ISSUE TITLE
[Bug]: Update to the latest version of the Facebook SDK for ioS for this app

### ISSUE THREAD

**Issue by mingV2019** [oddbit/flutter_facebook_app_events#363]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

<img width="382" alt="image" src="https://github.com/oddbit/flutter_facebook_app_events/assets/47814865/b0110115-9c46-4f2d-9c87-291904f88468">

<img width="349" alt="image" src="https://github.com/oddbit/flutter_facebook_app_events/assets/47814865/51f80504-5995-4352-a719-2b7c28bf8ea2">

### Expected Behavior

successsaasdasdasdasdasd

### Steps To Reproduce

noasdasdasdsadadasdasdasda

### Platform

iOS

### Flutter Doctor

```shell
nosadsadfsfsadasdasdsadasdasdasdasdasdasdasdasdasd
```

**Comment by yelkamel** [comment]
I have the same issue even if I added the SDKAdNetwork. 
**[https://developers.facebook.com/docs/setting-up/platform-setup/ios/SKAdNetwork](https://developers.facebook.com/docs/setting-up/platform-setup/ios/SKAdNetwork)**

```
<key>SKAdNetworkItems</key>
  <array>
    <dict>
      <key>SKAdNetworkIdentifier</key>
      <string>v9wttpbfk9.skadnetwork</string>
    </dict>
    <dict>
      <key>SKAdNetworkIdentifier</key>
      <string>n38lu8286q.skadnetwork</string>
    </dict>
  </array>
</dict>
```


But I don't think this is related to this package.

@mingV2019  you can try by adding this to your info.plist

**Comment by antonoddbit** [comment]
@mingV2019 please recheck you project settings, the latest version released in last month (0.19.2) is already using the latest FBSDK version. @yelkamel thanks for adding the plist info

**Comment by HarisHare** [comment]
I've had the same issue on GitHub Actions. I specified the XCode version to 15.3 and it worked. I've used [this action](https://github.com/marketplace/actions/setup-xcode-version) to specify the version if anyone finds it helpful.
